### PR TITLE
EES-1405 Fix zip case handling issues.

### PR DESCRIPTION
**Pull request by shakes90**

### CODE REVIEW

File: src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs [hunk 1]
```diff
@@ -44,11 +44,11 @@ await using (var stream = dataFile.Open())
                     releaseId: releaseId,
                     stream: stream,
                     fileType: ReleaseFileTypes.Data,
-                    fileName: dataFile.Name,
+                    fileName: dataFile.Name.ToLower(),
                     contentType: "text/csv",
                     metaValues: GetDataFileMetaValues(
                         name: blob.Name,
-                        metaFileName: metadataFile.Name,
+                        metaFileName: metadataFile.Name.ToLower(),
```
Comment (ntsim) [inline src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs:51]: No need to do this as the `GetDataFileMetaValues` already lower cases the meta file name

File: src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs [hunk 2]
```diff
@@ -62,10 +62,10 @@ await using (var stream = metadataFile.Open())
                     releaseId: releaseId,
                     stream: stream,
                     fileType: ReleaseFileTypes.Metadata,
-                    fileName: metadataFile.Name,
+                    fileName: metadataFile.Name.ToLower(),
                     contentType: "text/csv",
                     metaValues: GetMetaDataFileMetaValues(
-                        dataFileName: metadataFile.Name,
+                        dataFileName: dataFile.Name.ToLower(),
```
Comment (ntsim) [inline src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs:68]: No need to do this as the `GetMetaDataFileMetaValues` already lower cases the data file name

File: src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs [hunk 3]
```diff
@@ -44,11 +44,11 @@ await using (var stream = dataFile.Open())
                     releaseId: releaseId,
                     stream: stream,
                     fileType: ReleaseFileTypes.Data,
-                    fileName: dataFile.Name,
+                    fileName: dataFile.Name.ToLower(),
                     contentType: "text/csv",
                     metaValues: GetDataFileMetaValues(
                         name: blob.Name,
```
Comment (ntsim) [inline src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs:50]: Thinking about it, shouldn't this be the subject name?

**Review by ntsim — CHANGES_REQUESTED** [review]

File: src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs (hunk 1 above)
Comment (shakes90) [reply src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs]: true - reverted

File: src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs (hunk 2 above)
Comment (shakes90) [reply src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs]: reverted

File: src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs (hunk 3 above)
Comment (shakes90) [reply src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs:50]: No that's OK - The subject is set on this line...
name: blob.Name

**Review by ntsim — APPROVED** [review]

**Pull request merged**